src/api/spec/jobs/event_notify_backend_job_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+# WARNING: If you change tests make sure you uncomment this line
+# and start a test backend. Some of the actions
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe EventNotifyBackendJob, type: :job, vcr: true do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    let!(:project) { create(:project) }
+    let(:response) { double(:response, body: '<response code="ok"/>') }
+
+    before do
+      Event::CreateProject.create(project: project.name)
+      allow(Backend::Connection).to receive(:post).and_return(response)
```

Comment:
The same thing that happened before this change. The event is marked as `queued: true` before it tries to notify the backend, so if that fails then the backend will never be notified. This needs to be fixed but this is part of the bigger changes we are making to the events system.